src/pages/Summary/index.tsx

```diff
@@ -157,6 +174,60 @@ const _SummaryPage = ({
           />
         </span>
       </AcknowledgementSection>
+      <AcknowledgementSection title="Protect yourself against double deposits">
+        {beaconChainApiStatus === BeaconChainStatus.DOWN && (
+          <Alert variant="warning" className="mb20">
+            <div className="flex">
+              <AlertIcon />
+              <Text
+                weight={500}
+                color="yellowDarkest"
+                className="my10"
+                style={{ wordBreak: 'break-word' }}
+              >
+                Proceed with caution. Our on-chain data source is down and we
+                are unable to flag any double deposits.
+              </Text>
+            </div>
+          </Alert>
+        )}
+
+        <Text>
+          Again, <i>you are responsible for this transaction!</i> Duplicate
+          deposits with the same keyfile public key will not be rejected, but
+          your funds will be lost.
```

Comment:
Technically, the user can withdraw in phase 1.5 or phase 2. Maybe change the wording to something like:

```suggestion
          Again, <i>you are responsible for this transaction!</i> Duplicate
          deposits with the same keyfile public key will be considered as a
          top-up and the extra balance more than {PRICE_PER_VALIDATOR} {TICKER_NAME}
          will NOT be counted in your effective balance on the beacon chain.
```